chapter_13_database_layer_validation.asciidoc

```diff
@@ -201,9 +213,12 @@ Let's hack it in to see it work:
 // Which results in raising the correct error. But I think we should always
 // show the correct order even when it doesn't matter: validation first and
 // only then attempting save().
+// DAVID: A bit like Csanad, why bother calling item.save() at all?
 
 That gets the unit test to pass:
 
+// DAVID: Would be good to provide the command to run the unit test.
```

Comment:
i actually gave up on reminding people how to run the unit tests back in chapter 5 or so.  we do it an awful lot!